<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>faultsandwarnings</class>
 <widget class="QMainWindow" name="faultsandwarnings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>191</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vl_fault2"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>191</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vl_fault3"/>
   </widget>
   <widget class="QPushButton" name="pb_close">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>10</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(237, 51, 59);
border-radius: 20px;
font: 18pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_name">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Ubuntu&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Faults and Warnings</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>70</y>
      <width>191</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vl_warnings1"/>
   </widget>
   <widget class="QLabel" name="l_logo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/images/warning.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>191</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vl_fault1"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>70</y>
      <width>191</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vl_warnings2"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
